docs/wxm-devices/wifi-m5-bundle/m5-maintenance.mdx

```diff
@@ -7,26 +7,27 @@ slug: /wxm-devices/wifi-m5-bundle/m5-maintenance
 
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
-The maintenance of the outdoor sensor is an important process, which should be conducted meticulously over the suggested period of time.
+The maintenance of the weather station is an important process, which should be conducted meticulously over the suggested period of time.
 This process includes the cleaning of certain parts of the outdoor unit and the replacement of batteries.
 The maintenance process is summarised in the following steps:
 
 <div class="explanation">
   <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/maintenance.svg")} />
   <div>
     <h4>Instructions for opening the rain gauge</h4>
-      Clean the rain gauge at least once every 3 months.
+      <li>Clean the rain gauge at least once every 3 months.</li>
       Rotate the funnel counter-clockwise and lift to expose the rain gauge mechanism
       and clean with a damp cloth. Remove any dirt, debris and insects.
       If bug infestation is an issue, spray the array lightly with insecticide.
       It is important not to swing the rain gauge spoon during this process,
       as the weather station will measure and record faulty precipitation.
-      In order to avoid faulty measurements, you may turn off the M5 miner until finishing the entire process.
+      <li>Clean the solar panel at least every 3 months with a damp cloth.</li>
+      <li>In order to avoid faulty measurements, you may turn off the M5 miner until finishing the entire process.</li>
```

Comment:
Please use **numbered** bullets here as we do in WS2000. You should enclose the list with `<ol>` and `</ol>` at the end.

For example the text on WS2000 is:

```
    <ol>
      <li>Clean the rain gauge at least once every 3 months.</li>
      Rotate the funnel counter-clockwise and lift to expose the rain gauge mechanism
      and clean with a damp cloth. Remove any dirt, debris and insects.
      If bug infestation is an issue, spray the array lightly with insecticide.
      It is important not to swing the rain gauge spoon during this process,
      as the weather station will measure and record faulty precipitation.
      <li>Clean the solar panel at least every 3 months with a damp cloth.</li>
      <li>Remove any dirt, debris and insects from the thermo-hygro sensor every 12 months. It may be needed to open the radiation shield using a screwdriver and fully clean the thermo-hygro sensor in case there is too much debris on it.</li>
    </ol>
```